app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [Person(name=human["name"], age=human["age"])
+               for human in people]
+    for human in people:
+        if human.get("wife"):
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif human.get("husband"):
+            person_husband = Person.people[human["husband"]]
```

Comment:
it is better to make a variable for Person.people[human["name"]] cause u are using it for 2 times in ur code